src/help/guides/getting-started.md

```diff
@@ -63,42 +64,42 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
    <el-tag type="warning">Not the Windows Store app!</el-tag>
 1. Download and install [iCloud for Windows 8](https://support.apple.com/en-us/HT204283).
    <el-tag type="warning">Not the Windows Store app!</el-tag>
-   > Can't find the link to download **iCloud** for Windows 8? [Image](https://imgur.com/a/P1ef4Wd)
-1. Restart your PC after installing both **iTunes** and **iCloud**.
+  > Can't find the link to download **iCloud** for Windows 8? [GIF](https://imgur.com/a/P1ef4Wd)
+1. Restart your PC after you've installed both **iTunes** and **iCloud**.
    <el-tag type="warning">You need to restart your computer!</el-tag>
 1. Extract the downloaded **`AltInstaller.zip`** file. Run **`setup.exe`** and follow the instructions.
-1. Launch **AltServer** from your *Notification Area*/*System Tray* or *Hidden Icons*.
+1. Launch **AltServer** from your *Notification Area*.
```

Comment:
Isn't *Hidden Icons* different from *Notification Area*?

Reply:
Both Hidden Icons & System Tray are 'legacy' naming. The area is called the _Notification Area_, and next to it is the _Action Centre_.

Since it's a Windows 10 only guide, I think it's best here to use Microsoft's proper naming.

Reply:
Ok, thanks. As I don't use Windows I had lots of difficulties to find the proper denomination, I will update the translation.